### ISSUE TITLE
ag-grid does not support custom date time filter

### ISSUE THREAD

**Issue by srikant21** [ag-grid/ag-grid#3051]
```
How to implement the custom date time filter??
The below link provides to implement the date filter
https://www.ag-grid.com/javascript-grid-date-component/
I followed this link and i have 2 below issue while implementing date time:-

1. How to get the datetime value at this parameter "filterLocalDateAtMidnight"?
2. How to hit the "filterparams" "comparator" function only after i enter the date and time?

```
**Providing a Reproducible Scenario**
Please find the plunkr which i created below
https://plnkr.co/edit/EXM1vNqsGh9x60BT5XmZ?p=preview

**Current behavior**

Only Date is getting filtered.

**Expected behavior**
Datetime filter should be supported.
While implementing the datetime filter facing 2 issues listed below
1. How to get the datetime value at this parameter "filterLocalDateAtMidnight"?
2. How to hit the "filterparams" "comparator" function only after i enter the date and time?

**Please tell us about your environment:**

Operating system: windows 10
IDE:- Visual studio code
Angular CLI-- 7.3.6

* **ag-Grid version:** 20.2.0

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
Typescript:- 3.2.4

**Comment by makinggoodsoftware** [comment]
Hi,

ag-grid staff does not provide support through github. If you are a customer please contact us through our official support channel. Note that this issue has been flagged as managed-by-the-community, we also recommend you to check stackoverflow, note that if there is no activity on this issue after a while we will close it.

Thanks

**Comment by srikant21** [comment]
I am a free customer . I checked in stackoverflow but there is no topic related to this.
Can you please provide a solution on this??

**Comment by srikant21** [comment]
Hi,Without giving a reason for won't fix label will not make the issue to go away.Kindly not give this type of reason and close the defect.It is something if you are free customer then there is no value.

**Comment by leifjones** [comment]
@srikant21 Hopefully this repo (and linked article) might be helpful for you

https://github.com/jharris711/DateTime-Filter-Component-For-AG-Grid
